package.json

```diff
@@ -16,6 +16,7 @@
     "next": "13.3.0",
     "react": "18.2.0",
     "react-dom": "18.2.0",
+    "socket.io": "^4.7.0",
```

Comment:
do you need this already? 
you dont need to deinstall but for next time: only install things when you use them.

Reply:
alright. this is just for a possible later use ;)